packaging/windows/installer.nsi

```diff
@@ -178,7 +178,7 @@ Function NetdataConfigPage
 
         ${NSD_CreateLabel} 0 75% 20% 10% "URL"
```

Comment:
Cloud URL

Reply:
I pushed a commit with this suggestion. Thanks!